src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/CSharpFormatter.cs

```diff
@@ -72,16 +72,9 @@ internal class CSharpFormatter
                 return Array.Empty<TextEdit>();
             }
 
-            TextEdit[] edits;
-            if (formatOnClient)
-            {
-                edits = await FormatOnClientAsync(context, projectedRange, cancellationToken);
-            }
-            else
-            {
-                edits = await FormatOnServerAsync(context, projectedRange, cancellationToken);
-            }
-
+            var edits = formatOnClient
```

Comment:
I feel like in this particular case ternary looks good, but I've noticed it being suggested in places where it actually makes the code look much messier, particularly in the case of nested-ifs. I'm wondering if it might be OK to just make the ternary suggestion silent instead of a hint? Open to other thoughts though.

Reply:
I agree that sometimes the ternary it suggests is too crazy to be considered an improvement, which is why I never apply that suggestion blindly. On the other hand the ternary is often a nice improvement and I'd hate to lose those suggestions. I wish we could set a "complexity threshold" after which the ternary suggestion is silent.